recipes/tf_object_detection/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "tf_object_detection" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/Maxar-Corp/ratpac/releases/download/od-{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 476eaf754c61c5afce39e36683bb4a7a6b7f9084e2bc89f075d58e294da27768
+
+build:
+  number: 0
+  skip: True  # [win]  # Skip all windows for now due to numpy/scipy version pinning issues
+  skip: True  # [win32]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tensorflow >=1.12.0
+    - numpy >=1.16.1
+    - protobuf >=3.0.0
+    - jupyter >=1.0.0
+    - lxml
+    - matplotlib
+    - cython
+    - contextlib2
+
+test:
+  imports:
+    - tf_object_detection  # [win]
```

Comment:
you are skipping win, and we should have a test for linux and mac.  Or maybe this `# [win]` selector shouldn't be here?

Reply:
oh yeah. it should be removed!

Reply:
@scopatz on the recipe for [tensorflow-feedstock](https://github.com/conda-forge/tensorflow-feedstock/blob/master/recipe/meta.yaml), testing on `linux` and `OSX` has been is indicated to be challenging. The comments section says:

       Skip the import test on Linux as wheel file require a more recent
       version of GLIBC++ than the VM used to build and test package.
       Also skip on OSX due to OSX version requirements, see
       https://github.com/conda-forge/tensorflow-feedstock/pull/62

Reply:
@conda-forge/tensorflow I'm building recipe for the object_detection API by following the steps on `tensorflow-feedstock` recipe. please help us understand the issue with skipping test for `linux` and `osx`.

Reply:
The `tensorflow` packages on conda-forge are repackage versions of the PyPI wheels provided by Google.  Despite being labeled as `manylinux1` these wheels do not meet the manylinux specification.  Specifically the wheels and conda packages require Ubunutu 16.04 and must be run on a processor with the AVX instruction set.  Therefore they will not work on the CentOS 6 image that is used for building and testing on conda-forge.  More details on this can be found in conda-forge/tensorflow-feedstock#6.  Building Tensorflow from source would produce packages which are compatible with CentOS 6 but this process is very time consuming, see conda-forge/tensorflow-feedstock#69, if anyone wants to work on this help would be appreciated.

Reply:
@jjhelmus thanks. I hope that will help @scopatz understand the issue here.